### PR TITLE
feat(map): Advanced map tool

### PR DESCRIPTION
**Pull request by almat65**
**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines:https://github.com/infra-geo-ouverte/igo2/blob/master/.github/CONTRIBUTING.md#git-commit-guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** 
No advenced tools

**What is the new behavior?**
2 tools available. "Swipe" for compare the layers.
"Coordinates" for use the coordinates of the center of the map in differents projections. Display a cuttent zoom-level

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x ] No

**If this PR contains a breaking change, please describe the impact and migration path for existing applications:**

**Other information**:
image

### CODE REVIEW

File: packages/integration/src/lib/map/advanced-map-tool/advanced-coordinates/advanced-coordinates.component.ts [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+  /**
+   * Listen a state of the map, a state of a form, update the coordinates
+   */
+  ngOnInit(): void {
+    this.mapState$$ = combineLatest([this.map.viewController.state$.pipe(debounceTime(50)), this.form.valueChanges])
+        .subscribe(() => {
+      this.currentCenterDefaultProj = this.map.viewController.getCenter(this.defaultProj.code);
+      const currentMtmZone = zoneMtm(this.currentCenterDefaultProj[0]);
+      const currentUtmZone = zoneUtm(this.currentCenterDefaultProj[0]);
+      if (!this.inQuebec && currentMtmZone !== this.currentZones.mtm) {
```
Comment (pelord) [inline packages/integration/src/lib/map/advanced-map-tool/advanced-coordinates/advanced-coordinates.component.ts:75]: maybe rename inquebec by inMtmZone

Comment (almat65) [reply packages/integration/src/lib/map/advanced-map-tool/advanced-coordinates/advanced-coordinates.component.ts:75]: ok

**Comment by pelord** [comment]
https://github.com/infra-geo-ouverte/igo2/pull/658

File: packages/integration/src/locale/fr.integration.json [hunk 2]
```diff
@@ -64,7 +65,34 @@
         "maximizeTooltip": "Agrandir la fenêtre",
         "standardExtent": "Réduire",
         "standardExtentTooltip": "Réduire la fenêtre"
-      }
+      },
+      "advanced-map-tool": {
+        "advanced-swipe":{      
+          "swipe-select": "Choisir les couches à swiper",
```
Comment (pelord) [inline packages/integration/src/locale/fr.integration.json:71]: Change "swiper" by "comparer"

**Pull request merged**